README.md

```diff
@@ -163,3 +163,12 @@ The above requirements are in the supplied Brewfile and installed via `scripts/s
 * `scripts/setup.sh`
 * `nvm use`
 * `npm run build`
+
+Testing Changes and Observability
+---------------------------------
+
+Due to the nature of this project, there is no pre-production environment available for testing changes. Consequently, we recommend using Riff-Raff to deploy your branch to an individual account in order to validate your changes in production. In order to do this, select `Preview` from the deployment page (instead of `Deploy Now`). Next `Deselect all` and then manually select all deployment tasks for a specific account. Once you’ve done this you can `Preview with selections`, check the list of tasks and then `Deploy`.
```

Comment:
Is it worth adding a version of this list to the PR template too? (You might be able to gauge my level of anxiety towards this repository now 😄 ).

Reply:
> Is it worth adding a version of this list to the PR template too?

Good idea. I've added a PR template to this branch.